src/app/tests/suites/OTA_SuccessfulTransfer.yaml

```diff
@@ -0,0 +1,177 @@
+        defaultValue: 5560
+    providerKvs:
+        type: CHAR_STRING
+        defaultValue: "/tmp/chip_kvs_provider"
+    otaImageFilePath:
+        type: CHAR_STRING
+        defaultValue: "/tmp/otaImage"
+    rawImageFilePath:
+        type: CHAR_STRING
+        defaultValue: "/tmp/rawImage"
+    rawImageContent:
+        type: CHAR_STRING
+        defaultValue: "Have a hootenanny!"
+    downloadImageFilePath:
+        type: CHAR_STRING
+        defaultValue: "/tmp/downloadedImage"
+
+tests:
+    - label: "Create OTA image"
+      cluster: "SystemCommands"
+      command: "CreateOtaImage"
+      arguments:
+          values:
+              - name: "otaImageFilePath"
+                value: otaImageFilePath
+              - name: "rawImageFilePath"
+                value: rawImageFilePath
+              - name: "rawImageContent"
+                value: rawImageContent
+
+    - label: "Start the provider with an image"
+      cluster: "SystemCommands"
+      command: "Start"
+      arguments:
+          values:
+              - name: "registerKey"
+                value: "chip-ota-provider-app"
+              - name: "discriminator"
+                value: providerDiscriminator
+              - name: "port"
+                value: providerPort
+              - name: "kvs"
+                value: providerKvs
+              - name: "filepath"
+                value: otaImageFilePath
+
+    - label: "Commission the provider from alpha"
+      identity: "alpha"
+      cluster: "CommissionerCommands"
+      command: "PairWithQRCode"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: providerNodeId
+              - name: "payload"
+                value: providerPayload
+
+    - label: "Wait for the commissioned provider to be retrieved for alpha"
+      identity: "alpha"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: providerNodeId
+
+    - label: "Install ACL for QueryImage"
+      cluster: "Access Control"
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value: [
+                  # Grant administer privilege to the default controller node ID 112233
+                  {
+                      FabricIndex: 1,
+                      Privilege: 5,
+                      AuthMode: 2,
+                      Subjects: [112233],
+                      Targets: null,
+                  },
+                  # Grant operate privileges to all nodes for the OTA Provider cluster on every endpoint
+                  {
+                      FabricIndex: 1,
+                      Privilege: 3,
+                      AuthMode: 2,
+                      Subjects: null,
+                      Targets: null,
+                      [{ Cluster: 41, Endpoint: null, DeviceType: null }],
+                  },
+              ]
+
+    - label: "Stop the requestor"
```

Comment:
Isn't this "Stop the provider" ?

Reply:
No, this is stopping the requestor, AKA the one with "default" register key